scipy/signal/spectral.py

```diff
@@ -26,30 +27,34 @@ def periodogram(x, fs=1.0, window=None, nfft=None, detrend='constant',
     x : array_like
         Time series of measurement values
     fs : float, optional
-        Sampling frequency of the `x` time series. Defaults to 1.0.
+        Sampling frequency of the ``x`` time series. Defaults to 1.0.
```

Comment:
I kind of like how this looks in the sphinx docs, though. ([circleCI](https://1587-1460385-gh.circle-artifacts.com/0/home/ubuntu/scipy/doc/build/html-scipyorg/generated/scipy.signal.periodogram.html#scipy.signal.periodogram) vs [current](https://docs.scipy.org/doc/scipy/reference/generated/scipy.signal.periodogram.html#scipy.signal.periodogram))

Take the `detrend` argument for example, it makes it more clear when we're referring to the function by the name of "detrend" and the keyword argument named "detrend".